contracts/multisig-prover/src/encoding/bcs.rs

```diff
@@ -33,8 +33,8 @@ pub fn make_operators(worker_set: WorkerSet) -> Operators {
 pub fn transfer_operatorship_params(worker_set: &WorkerSet) -> Result<HexBinary, ContractError> {
     let mut operators: Vec<(HexBinary, Uint256)> = worker_set
         .signers
-        .iter()
-        .map(|s| (s.pub_key.clone().into(), s.weight))
+        .values()
+        .map(|s| (s.1.clone().into(), s.0))
```

Comment:
```suggestion
        .map(|(weight,pub_key)| (...))
```

Reply:
Why `...`?

Reply:
I just didn't feel like typing out the rest of it, it's not proper syntax